apps/extension/src/utils/taskMasterApi.ts

```
@@ -0,0 +1,1347 @@
+			this.cacheAnalytics.evictions++;
+		}
+
+		if (entries.length > 0) {
+			console.log(`TaskMasterApi: Evicted ${entries.length} LRU cache entries`);
+		}
+	}
+
+	/**
+	 * Estimate data size for cache analytics
+	 */
+	private estimateDataSize(data: any): number {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)